modules/migration-upgrading-migration-tool.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+// * migration/migrating_3_4/deploying-cam-3-4.adoc
+// * migration/migrating_4_1_4/deploying-cam-4-1-4.adoc
+// * migration/migrating_4_2_4/deploying-cam-4-2-4.adoc
+[id='migration-upgrading-migration-tool_{context}']
+= Upgrading the Cluster Application Migration Tool
+
+You can upgrade your Cluster Application Migration (CAM) tool on your source and target clusters.
+
+If you are upgrading from CAM 1.1 to 1.2, you must update the service account token in the CAM web console.
+
+[id='upgrading-cam-ocp-4_{context}']
+== Upgrading the CAM on an {product-title} 4 cluster
+
+You can upgrade CAM on an {product-title} 4 cluster with the Operator Lifecycle Manager.
+
+If you selected the *Automatic* approval option when you subscribed to the Cluster Application Migration Operator, the CAM is updated automatically.
```

Comment:
I would remove "the" or change this to "the CAM **tool**..."

Reply:
> LGTM. A few quick nits.
> 
> What do you think about "Upgrading" info being in a section titled "Deploying CAM"? I would update the assembly title to "Deploying and **upgrading**...", to give a little extra context, but you can make the call.

Only problem is that I will be adding a similar section to 4.5 and I don't want to deal with merge conflicts. Oh, well, I guess I can deal with that when it happens.